internal/portforward/service/start.go

```diff
@@ -69,6 +69,14 @@ func (s *Service) Start(ctx context.Context) (runError <-chan error, err error)
 		return nil, fmt.Errorf("writing port file: %w", err)
 	}
 
+	if s.settings.Scriptpath == "" {
```

Comment:
This needs to change anyway, but just in case ™️ it should be

```suggestion
	if s.settings.Scriptpath != "" {
```